mappings/net/minecraft/network/packet/c2s/play/PlayerInteractEntityC2SPacket.mapping

```diff
@@ -3,8 +3,23 @@ CLASS net/minecraft/class_2824 net/minecraft/network/packet/c2s/play/PlayerInter
 	FIELD field_12870 entityId I
 	FIELD field_12871 type Lnet/minecraft/class_2824$class_2825;
 	FIELD field_12872 hitPos Lnet/minecraft/class_243;
+	FIELD field_25660 playerSneaking Z
+	METHOD <init> (Lnet/minecraft/class_1297;Lnet/minecraft/class_1268;Lnet/minecraft/class_243;Z)V
+		ARG 1 entity
+		ARG 2 hand
+		ARG 3 hitPos
+		ARG 4 playerSneaking
+	METHOD <init> (Lnet/minecraft/class_1297;Lnet/minecraft/class_1268;Z)V
+		ARG 1 entity
+		ARG 2 hand
+		ARG 3 playerSneaking
+	METHOD <init> (Lnet/minecraft/class_1297;Z)V
+		ARG 1 target
```

Comment:
Why not entity like the other constructors?

Reply:
probably because there are two entities, the player and the interacted entity

Reply:
Bacause there is 3 constructors and it differs the type of the interaction by the parameters.
example: first constuctor is for ATTACK (therefore "target"), second and third are INTERACT and INTERACT_AT respectively